frontend/src/components/HomePage/Middle/StoriesHomeFeed.js

```diff
@@ -17,7 +23,40 @@ function StoriesHomeFeed() {
     (state) => state.simpleUsers[sessionUser.id]?.profile_picture
   );
 
-  if (Object.keys(stories).length > 3) {
+  let currentStories;
+
+  if (currentWindow === 0) {
+    currentStories = Object.values(stories).slice(
+      currentWindow,
+      currentWindow + 3
+    );
+  } else {
+    currentStories = Object.values(stories).slice(
+      currentWindow,
+      currentWindow + 4
+    );
+  }
+
+  const moveLeft = (e) => {
+    if (e) {
+      e.preventDefault();
+    }
```

Comment:
shouldn't need this check right?
```suggestion
      e.preventDefault();
```

Reply:
Do I have access to the the event object if I call it from my useEffect? Pretty sure I can't pass it there but I know I didn't check that. I put this here because I call this function from my setInterval